pkg/landscaper/jsonschema/testreg/cert.go

```diff
@@ -125,7 +125,7 @@ func GenerateCertificates() (*Certificate, error) {
 const chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
 
 func init() {
-	mathrand.Seed(time.Now().Unix())
+	mathrand.New(mathrand.NewSource(time.Now().Unix()))
```

Comment:
Why this change? I do not understand the construct at all. The generated Rand object is never used.

Reply:
Linter complained after a dependency upgrade that `Seed()` is deprecated. Replaced it with the suggestion provided within the library, but you are right, it is never used. Removed it.